inst/extdata/gcam-usa/UCS_tech_names.csv

```diff
@@ -1,3 +1,6 @@
+# File: UCS_tech_names.csv
```

Comment:
should we define UCS in the code or in the CSV files?

Reply:
done :+1: